src/openrct2/actions/MazeAction.h

```diff
@@ -0,0 +1,14 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2022 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "GameAction.h"
+
+money64 MazeCalculateCost(money32 constructionCost, const Ride& ride, const CoordsXYZ& _loc);
```

Comment:
This entire header only contains one function, and nothing relates to it being an action. This would better fit in `src/openrct2/ride/gentle/Maze.cpp` for example.

Reply:
I moved the code and also updated the network version while I was at it.